src/app.py

```
@@ -0,0 +1,87 @@
+    """Generate a math question using the AI model"""
+    try:
+        # Add a random seed to prevent caching
+        random_seed = random.randint(1, 10000)
+
+        response = openai.chat.completions.create(
+            model=MODEL_NAME,
+            temperature=1.0,
+            messages=[
+                {
+                    "role": "system",
+                    "content": "Generate a simple math question randomly. Provide the question and correct answer in the format: 'QUESTION: <question>\nANSWER: <answer>'"
```

Pylint (3.3.7)

[convention] Line too long (170/100)

(C0301)